toolset/travis/travis_long.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Use to generate a new .travis.yml that runs individual
+# tests within a framework directory in a new vm
+
+# Backup original travis file
+mv $FWROOT/.travis.yml $FWROOT/.travis.bak
+
+# generate new matrix
+MATRIX=`$FWROOT/toolset/run-tests.py --list-tests | sed '/FWROOT */d' | sed '/Time */d' | sed '/Results */d' | sed -E 's/(.+)/    - "TEST=\1"/g'`
+
+tee $FWROOT/.travis.yml <<EOF
+sudo: required
+dist: trusty
+language: generic
+python:
+  - "2.7"
+  
+env:
+  matrix:
+$MATRIX
+
+before_script:
+  - source ./toolset/travis/travis_clean.sh
+
+script:
+  - tfb --mode verify --test "\$TEST"
```

Comment:
this **might** have to be `- ./toolset/run-tests.py --mode verify --test "\$TEST"`
the `tfb` command should be available by now.